sdcm/sct_events/setup.py

```diff
@@ -129,7 +129,7 @@ def enable_default_filters(sct_config: SCTConfiguration):  # pylint: disable=unu
     # issue that should be track https://github.com/scylladb/scylla-enterprise/issues/3148
     EventsSeverityChangerFilter(new_severity=Severity.WARNING,
                                 event_class=CassandraStressLogEvent.ConsistencyError,
-                                regex="Authentication error on host.*Cannot achieve consistency level for cl ONE").publish()
+                                regex=r".*Authentication error on host.*Cannot achieve consistency level for cl ONE").publish()
```

Comment:
possibly just adding "r" is enough. If we dropped first .* then it would be faster

Reply:
it wasn't enough, I've checked